pkg/controller/kieapp/constants/constants.go

```diff
@@ -76,23 +76,23 @@ var VersionConstants = map[string]*api.VersionConfigs{
 		BrokerImage:      "amq-broker-73-openshift",
 		BrokerImageTag:   "7.3",
 		DatagridImage:    "datagrid73-openshift",
-		DatagridImageTag: "1.1",
+		DatagridImageTag: "1.2",
 	},
 	LastMicroVersion: {
 		APIVersion:       v1.SchemeGroupVersion.Version,
 		ImageTag:         "1.1",
 		BrokerImage:      "amq-broker-73-openshift",
 		BrokerImageTag:   "7.3",
 		DatagridImage:    "datagrid73-openshift",
-		DatagridImageTag: "1.1",
+		DatagridImageTag: "1.2",
```

Comment:
looks like you're targeting only version 7.5.0 for this, but with this imagetag change... you're affecting 7.4.1 as well. You should likely only modify the "CurrentVersion" mapping, as that reflects 7.5.0.
https://github.com/kiegroup/kie-cloud-operator/blob/32ea6118a75f11c9d48362578127608f10e8efa6/pkg/controller/kieapp/constants/constants.go#L9-L16

Reply:
Got it; thanks. I'll force-push the fix. Much appreciated. :)